packages/sources/react/src/components/actions/VtmnDropdown/VtmnDropdown.tsx

```diff
@@ -46,6 +55,20 @@ export const VtmnDropdown = ({
           role="group"
           aria-labelledby={props['id']}
         >
+          {/**
+           * Renders every child and inject onChange in VtmnDropdownItem
+           */}
+          {React.Children.map(children, (child: React.ReactNode) =>
+            React.isValidElement(child) && child.type === VtmnDropdownItem
+              ? React.cloneElement(child, {
+                  ...child.props,
+                  onChange: (ev: React.ChangeEvent<HTMLDivElement>) => {
+                    onChange && onChange(child.props.value);
+                    child.props.onChange && child.props.onChange(ev);
+                  },
+                })
+              : child,
+          )}
           {children}
```

Comment:
Why do we still keep this line `{children}` if you already map on children thanks to `React.Children.map`?
Thanks for your help!